packages/undo-manager/src/sync-undo-manager.ts

```diff
@@ -0,0 +1,44 @@
+/**
+ * WordPress dependencies
+ */
+import { getSyncProvider } from '@wordpress/sync';
+
+/**
+ * Internal dependencies
+ */
+import type { UndoManager, HistoryRecord } from './types';
+
+/**
+ * Creates the sync powered undo manager.
+ *
+ * @return {UndoManager} Undo manager.
+ */
+export function createSyncUndoManager() {
```

Comment:
Prefer types over docblock statements

```suggestion
 */
export function createSyncUndoManager(): UndoManager {
```

Reply:
Good point, I forgot to do that during my conversion of this package to typescript.